README.md

```diff
@@ -1,4 +1,4 @@
-# Lozad.js [![npm version](https://badge.fury.io/js/lozad.svg)](https://badge.fury.io/js/lozad)
+# Lozad.js [![npm version](https://badge.fury.io/js/lozad.svg)](https://badge.fury.io/js/lozad) [![Build Status](https://travis-ci.org/ApoorvSaxena/lozad.js.svg?branch=travis-integration)](https://travis-ci.org/ApoorvSaxena/lozad.js)
```

Comment:
Why you are pointing to the `travis-integration` branch? Pointing to the `master` would not be better?

Reply:
had fixed it later on master directly, as pointing to master before would have shown `build|failed` on repo for a few minutes till deployment